marklogic-data-hub-central/ui/src/components/modeling/property-modal/property-modal.tsx

```diff
@@ -226,6 +233,10 @@ const PropertyModal: React.FC<Props> = (props) => {
     }
   }, [props.isVisible]);
 
+
```

Comment:
Please remove these extra spaces here

Reply:
Thank you @mrodrigu-marklogic I could find the way to show the pointer and force the hover and focus effect in the input inside it too:
![image](https://user-images.githubusercontent.com/89084298/146599899-83be0df8-f7a7-4131-83e2-5497ddc58c61.png)